src/SharpRaven.sln

```diff
@@ -81,96 +97,136 @@ Global
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Debug 3.5|x86.ActiveCfg = Debug 3.5|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Debug 3.5|x86.Build.0 = Debug 3.5|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Debug 4.0|Any CPU.ActiveCfg = Debug 4.0|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Debug 4.0|Any CPU.Build.0 = Debug 4.0|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Debug 4.0|Mixed Platforms.ActiveCfg = Debug 4.0|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Debug 4.0|Mixed Platforms.Build.0 = Debug 4.0|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Debug 4.0|x86.ActiveCfg = Debug 4.0|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Debug 4.0|x86.Build.0 = Debug 4.0|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Debug 4.5|Any CPU.ActiveCfg = Debug 4.5|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Debug 4.5|Any CPU.Build.0 = Debug 4.5|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Debug 4.5|Mixed Platforms.ActiveCfg = Debug 4.5|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Debug 4.5|Mixed Platforms.Build.0 = Debug 4.5|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Debug 4.5|x86.ActiveCfg = Debug 4.5|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Debug 4.5|x86.Build.0 = Debug 4.5|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Release 3.5|Any CPU.ActiveCfg = Release 3.5|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Release 3.5|Any CPU.Build.0 = Release 3.5|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Release 3.5|Mixed Platforms.ActiveCfg = Release 3.5|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Release 3.5|Mixed Platforms.Build.0 = Release 3.5|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Release 3.5|x86.ActiveCfg = Release 3.5|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Release 3.5|x86.Build.0 = Release 3.5|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Release 4.0|Any CPU.ActiveCfg = Release 4.0|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Release 4.0|Any CPU.Build.0 = Release 4.0|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Release 4.0|Mixed Platforms.ActiveCfg = Release 4.0|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Release 4.0|Mixed Platforms.Build.0 = Release 4.0|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Release 4.0|x86.ActiveCfg = Release 4.0|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Release 4.0|x86.Build.0 = Release 4.0|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Release 4.5|Any CPU.ActiveCfg = Release 4.5|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Release 4.5|Any CPU.Build.0 = Release 4.5|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Release 4.5|Mixed Platforms.ActiveCfg = Release 4.5|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Release 4.5|Mixed Platforms.Build.0 = Release 4.5|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Release 4.5|x86.ActiveCfg = Release 4.5|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Release 4.5|x86.Build.0 = Release 4.5|Any CPU
```

Comment:
Here's the stuff that seems to have moved from [what is referenced in this comment](https://github.com/getsentry/raven-csharp/pull/156/files#r79582300).